.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
All files in the `.idea/` directory should not be merged into the repository. Please add `.idea/` into a `.gitignore` under the `flutter/` directory.

Reply:
These `.idea/` files are still being tracked. You should have a `.gitignore` in the root of this repo and add `.idea/` so that these files are not tracked.